web/templates/message_feed_errors.hbs

```diff
@@ -9,29 +9,59 @@
         {{/tr}}
     </p>
 </div>
-<div class="all-messages-search-caution hidden-for-spectators" hidden>
-    <p>
-        <i class="all-messages-search-caution-icon fa fa-exclamation-circle" aria-hidden="true"></i>
-        {{#tr}}
-        End of results from your
-        <z-link>history</z-link>.
-        {{#*inline "z-link"}}<a href="/help/search-for-messages#searching-shared-history"
-          target="_blank" rel="noopener noreferrer">{{> @partial-block}}</a>{{/inline}}
-        {{/tr}}
-        &nbsp;
-        <span>
+<div class="all-messages-search-caution banner-wrapper hidden-for-spectators" hidden>
```

Comment:
This is not the way we want to implementing banner anymore, the new banners are componentized and we can define them in JS. Please take a look at `web/src/navbar_alerts.ts` and `web/src/popup_banners.ts` to see how the other banners are being implemented.